### PR TITLE
Add --options-files flag to export to render templates

### PR DESCRIPTION
**Pull request by howardjohn**
Fixes https://github.com/GoogleCloudPlatform/k8s-cluster-bundle/issues/248

I don't know what is up with the go.mod changes, I could not build (even from master) without `go mod tidy` which introduces these changes

### CODE REVIEW

**Comment by howardjohn** [comment]
Bazel fails on deps but `bazel run //:gazelle -- update-repos -from_file=go.mod -to_macro 'repositories.bzl%gomod_repositories'` doesn't output anything, so a bit stuck here. Not a bazel expert

File: pkg/wrapper/bundlewrapper.go [hunk 1]
```diff
@@ -21,6 +21,8 @@ import (
 
 	bundle "github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/apis/bundle/v1alpha1"
 	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/converter"
+	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/options"
```
Comment (Kashomon) [inline pkg/wrapper/bundlewrapper.go:24]: These need to be added to the Bazel BUILD file

**Review by Kashomon — COMMENTED** [review]
This is the failure:

compilepkg: missing strict dependencies:
	/workspace/.bazel/sandbox/processwrapper-sandbox/263/execroot/__main__/pkg/wrapper/bundlewrapper.go: import of "github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/options"
	/workspace/.bazel/sandbox/processwrapper-sandbox/263/execroot/__main__/pkg/wrapper/bundlewrapper.go: import of "github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/options/multi"

File: pkg/wrapper/bundlewrapper.go (hunk 1 above)
Comment (howardjohn) [reply pkg/wrapper/bundlewrapper.go:24]: I thought gazelle was supposed to do it automatically? Either way, did it manually. Thanks

**Review by Kashomon — APPROVED** [review]

**Pull request merged**